Source/WebKit/NetworkProcess/Classifier/ResourceLoadStatisticsStore.h

```diff
@@ -440,6 +442,8 @@ class ResourceLoadStatisticsStore final : public DatabaseUtilities, public CanMa
     bool m_debugModeEnabled { false };
     bool m_dataRecordsBeingRemoved { false };
     bool m_isNewResourceLoadStatisticsDatabaseFile { false };
+
+    SharedPreferencesForWebProcess m_sharedPreferencesForWebProcess;
```

Comment:
This isn't what we want to check, this will just construct an empty object (so the feature will always be disabled).

`m_sharedPreferencesForWebProcess` on `NetworkConnectionToWebProcess` is what we need to check.